admin/.gitignore

```diff
@@ -6,3 +6,5 @@ social-income-prod-firebase-adminsdk.json
 /test-results/
 /playwright-report/
 /playwright/.cache/
+tests/playwright/sessionStorage.json
+tests/playwright/**/*darwin.png
```

Comment:
we add this so that we don't commit the screenshots generated on the mac? 👍

Reply:
yeah, exactly. those file never need to be comitted.